### PR TITLE
Add extra

### PR DESCRIPTION
**Pull request by pawelru**

### CODE REVIEW

File: .github/workflows/verdepcheck.yaml [hunk 1]
```diff
@@ -101,13 +106,14 @@ jobs:
 
       - name: Dependency Test - ${{ env.strategy }} 🔢
         id: verdepcheck
-        uses: insightsengineering/r-verdepcheck-action@main
+        uses: insightsengineering/r-verdepcheck-action@add_extra
```
Comment (pawelru) [inline .github/workflows/verdepcheck.yaml:109]: to revert
```suggestion
        uses: insightsengineering/r-verdepcheck-action@main
```

**Comment by pawelru** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/verdepcheck.yaml [hunk 2]
```diff
@@ -28,6 +28,11 @@ on:
           Strategy to test package dependencies. One of: min_isolate, min_cohort, release, max.
         required: true
         type: string
+      extra-deps:
+        description: Extra dependencies specified similarly as in the `DESCRIPTION` file, i.e. `"<package name> (<operator> <version>)"` where both `<operator>` and `<version>` are optional. Multiple entries are possible separated by `";"`.
```
Comment (walkowif) [inline .github/workflows/verdepcheck.yaml:32]:
```suggestion
        description: |
          Extra dependencies specified similarly as in the `DESCRIPTION` file,
          i.e. `"<package name> (<operator> <version>)"` where both `<operator>`
          and `<version>` are optional. Multiple entries are possible separated by `";"`.
```

File: .github/workflows/verdepcheck.yaml [hunk 3]
```diff
@@ -28,6 +28,11 @@ on:
           Strategy to test package dependencies. One of: min_isolate, min_cohort, release, max.
         required: true
         type: string
+      extra-deps:
+        description: Extra dependencies specified similarly to the `DESCRIPTION` file, i.e. `"<package name> (<operator> <version>)"` where both `<operator>` and `<version>` are optional. Multiple entries are possible separated by `";"`.
```
[the remark this answers is not in the document]
Comment (pawelru) [reply .github/workflows/verdepcheck.yaml]: done in 896d97951c4c7ad7dc4a5b009e8c625ab3587b9f

**Pull request merged**